src/main/resources/templates/findings/sarif.peb

```diff
@@ -44,7 +44,12 @@
             "group": "{{ finding.component.group }}",
             "version": "{{ finding.component.version }}",
             "source": "{{ finding.vulnerability.source }}",
-            "cwes": "{{ finding.vulnerability.cwes }}",
+            "cwes": [{% for cwe in finding.vulnerability.cwes %}
+                {
+                  "cweId": "{{ cwe.cweId }}",
+                  "name": "{{ cwe.name }}"
+                }
+            ]{% if not loop.last %},{% endif %}{% endfor %},
```

Comment:
I think this should handle commas within the array so needs to be within the `[]`.

```suggestion
                }
                {% if not loop.last %},{% endif %}{% endfor %}
            ],
```